sdks/go/pkg/beam/core/runtime/options.go

```diff
@@ -117,6 +118,23 @@ func (o *Options) Export() RawOptions {
 	return RawOptions{Options: copyMap(o.opt)}
 }
 
+// LoadOptionsFromFlags adds any flags not defined in excludeFlags to the options.
+// If the key is already defnined, it panics.
```

Comment:
```suggestion
// If the key is already defined, it panics.
```

Reply:
As I think about this more, I actually think this is probably not the behavior we want anyways. If a pipeline author tries to write to PipelineOptions with a value that also happens to be a flag, I think we want to let them do that - we shouldn't steamroll their value. I've updated to just ignore any keys that are already defined.